app/templates/components/forms/admin/settings/ticket-fees-form.hbs

```diff
@@ -87,7 +87,7 @@
     {{t 'Add New'}}
   </button>
 
-  <button class="ui teal button" type="submit">
+  <button class="ui teal button {{if model '' 'disabled'}}" type="submit">
```

Comment:
```hbs
{{unless model 'disabled'}}
```

Reply:
@CosmicCoder96 Made The changes. Please Review.